packages/modular-scripts/src/build.ts

```diff
@@ -199,59 +199,51 @@ async function makeBundle(
   const packageJson = packageJsonsByDirectoryName[directoryName];
 
   if (!packageJson) {
-    console.error(
+    throw new Error(
```

Comment:
I wasn't thinking this is an error, just a bypass. But I can see how people should be explicit in asking for the things to build, so yeah this should be an error.